### DISCUSSION TITLE
Update for Laravel 9

### DISCUSSION THREAD

**Discussion by code-poel** [ash-jc-allen/laravel-command-spinner#1] [Ideas]
Any chance you'll be updating the codebase to work with Laravel 9?

**Comment by ash-jc-allen** [comment]
Hey! This package was made as a bit of proof-of-concept and doesn't really tend to get any downloads, so I wasn't intending on updating it to work with Laravel 9.

But, if you'd like to make a PR to it to add support for Laravel 9, I'd be more than happy to review/accept one 😄
